index.html

```diff
@@ -13,13 +13,19 @@
   <!-- Coding the toolbar section of the page-->
   <header class="header-container">
     <a class="mylogo" href="#">My Logo</a>
-    <nav class="menu">
-      <img src="icons/menu.png" alt="Menu Icon">
+    <img class="mobile-menu" src="icons/menu.png" alt="Menu Icon">
+    <nav class="desktop-menu">
+      <ul class="menu-items">
+        <li class="menu-item"> <a href="#portfolio">Portfolio</a></li>
+        <li class="menu-item"> <a href="#about">About</a></li>
+        <li class="menu-item"> <a href="#contact">Contact</a></li>
+
+      </ul>
     </nav>
   </header>
   <section class="home">
     <section class="bio">
```

Comment:
- [ ] In my opinion, it would be better to make your website responsive as we are still between 375px and 1440. As you can see below :point_down: the background image is not responsive
![Screenshot from 2023-03-02 17-31-47](https://user-images.githubusercontent.com/97834160/222476866-953b7903-ede3-4aa9-874f-b4113b419787.png)